src/risedevtool/src/task/utils.rs

```diff
@@ -118,6 +118,10 @@ pub fn add_hummock_backend(
                 cmd.arg("--state-store")
                 .arg(format!("hummock+hdfs://{}", opendal.namenode));
             }
+            else if opendal.engine == "opendal_s3"{
+                cmd.arg("--state-store")
+                .arg(format!("hummock+opendal_s3://{}", opendal.bucket));
+            }
```

Comment:
This change is unnecessary? I don't think `hummock+opendal_s3` is vald.

Reply:
Yes, it's some out of date code, will remove.